test/modules/sungeun/init/printModuleInitOrder.na-none.good

```diff
@@ -46,6 +46,5 @@ Initializing Modules:
     ChapelError
     Builtins
     AlignedTSupport
-    CPtr
```

Comment:
I would've expected CPtr and SysError to move somewhere else in this file, not to just disappear.  Is it clear to you why it would change in this way?

Reply:
(specifically, I would think that even though they're not used by default in the user code anymore, their use in so many internal modules would still cause them to be initalized?)

Reply:
Right. Because our module initialization printing is not printing all module initializations. We have it controlled by `config const printModuleInitOrder = false;`. So until if we initialize a module before we initialize the module `PrintModuleInitOrder` we miss those in this output. With this PR CPtr moves that category. If I remove checking on that `config const`, I get the following output. Note that we are also not printing out initialization of bunch of other modules.

```
  PrintModuleInitOrder
   ChapelBase
    SysCTypes
    CPtr
     SysBasic
     SysError
     HaltWrappers
    ChapelTaskData
    String
     ByteBufferHelpers
     BytesStringCommon
      NVStringFactory
      Bytes
     CString
     StringCasts
Initializing Modules:   ### we never see stuff before this line
   ChapelStandard
    OwnedObject
    Atomics
     MemConsistency
    ChapelThreads
    ChapelTuple
     DSIUtil
    ChapelRange
    ChapelTaskDataHelp
    LocaleModel
     LocaleModelHelpFlat
      LocaleModelHelpSetup
       ChapelLocale
       DefaultRectangular
        ChapelArray
         ChapelDebugPrint
          IO
           FormattedIO
            Regexp
         ChapelPrivatization
         Sort
          List
           ChapelLocks
          MSBRadixSort
           RadixSortHelp
           ShellSort
          QuickSort
           ShallowCopy
           InsertionSort
        ChapelDistribution
         ChapelHashtable
        ExternalArray
        RangeChunk
        ChapelHashing
       ChapelNumLocales
       Sys
      LocaleModelHelpRuntime
     LocaleModelHelpMem
    LocalesArray
    ChapelIO
    LocaleTree
    ChapelTaskTable
    MemTracking
    ChapelUtil
    ChapelError
    Builtins
    AlignedTSupport
   printModuleInitOrder
    moduleInitTest
```

Reply:
OK, thanks for the explanation order.  I forgot about that flag (if I ever understood it).